### DISCUSSION TITLE
Replaceblock cached?

### DISCUSSION THREAD

**Discussion by klaidoshka** [Zrips/CMI#4973] [Enhancement]
Hello, I'm here just to ask a few questions about `/replaceblock` and suggest one thing 😅

So, I'm preparing a server for 1.16.5, trying to remove all the spawners, elytras, mending books, ... from the worlds (have to no matter what!). ⚠

Server is on my localhost, I tried to replace all the spawners with air (`/replaceblock id mob_spawner w air r g`), but had a need to stop the server, so I tried to use sub-commands (that were mentioned in the CMI wiki) to pause the replaceblock function, but instead, another replaceblock function started with default radius of 25 chunks... 🤔 I believe it was searching for "pause" blocks, no idea, I used it like this: `/replaceblock pause`. ANYWAYS, in the end my already-happening function of removing spawners didn't pause (or at least I think so, because tps was dropping down as usual).  Because of that I had to stop the server and today I tried to continue, but seems that every check is going right from the ZER0, ETA: 1.5days to finish... 😫

**In summary**: Would it be possible to get a new feature for /replaceblock ? To cache the started replaceblock action into some sort of data to continue later on, even if server was closed. Maybe it can be done by filling some sort of chunk numbers? I don't know 🙁 Ah, and do those sub-commands (pause/stop/continue) actually work? Because I tried them and nothing happened. Speed/AutoSpeed/Messages work indeed!

**Server version**: Yatopia 1.16.5 (I believe this would occur even if I used Spigot or Paper 👀)
**CMI version**: 8.8.0.6

**edit:** This cache thing would be useful probably for server-starters the most... Or who knows, maybe even for creating event-worlds, to replace something in wide range for long term of time with possibility to restart the server.
